src/arcade/potts/env/location/Plane.java

```diff
@@ -104,4 +104,46 @@ public int hashCode() {
         hash = 31 * hash + Double.hashCode(unitNormalVector.getZ());
         return hash;
     }
+
+    /**
+     * Rotates a given vector around a given axis by a given angle.
+     *
+     * <p>Rotation is performed using Rodrigues' rotation formula.
+     *
+     * @param vector the vector
+     * @param axis the axis of rotation
+     * @param thetaDegrees the angle of rotation in degrees
+     * @return the rotated vector scaled to unit length
+     */
+    public static Double3D rotateVectorAroundAxis(
+            Double3D vector, Direction axis, double thetaDegrees) {
```

Comment:
Since this is in the `Plane` class, is the idea that you would separately have the plane normal vector defined, then call this to get a rotated vector, then create a new `Plane` object? Or, does a `Plane` object already exist, and then you could call it to return a rotated version?

Not sure how this is being used, but to consider: instead of a `static` method here, you have a `public Plane rotate(Direction axis, double thetaDegrees)` that rotates the `this.unitNormalVector` and returns a new `Plane`

Reply:
yea I considered that as well, however currently I think the static method is better because I don't instantiate a "default" plane of division, I access the "default" plane of division's normal vector from the direction enum and then rotate that using this function before instantiating the plane.